superset-frontend/src/explore/components/DataTablesPane.tsx

```diff
@@ -191,11 +196,13 @@ export const DataTablesPane = ({
       if (data[type]?.length === 0) {
         return <span>No data</span>;
       }
+      const renderWithPagination = true;
       return (
         <TableView
           columns={columns[type]}
           data={filteredData[type]}
-          withPagination={false}
+          withPagination={renderWithPagination}
```

Comment:
Perhaps we don't need the `const` above for this, just do `true` here.

Reply:
Lint test is going to throw: `Value must be omitted for boolean attributes`

Reply:
Right, it should just be `withPagination` if it's set to true.